models/reddit_ads.yml

```diff
@@ -15,10 +15,10 @@ models:
             - status
             - time_zone_id
     columns:
-      - name: date_day
-        description: '{{ doc("date_day") }}'
-      - name: account_id
-        description: '{{ doc("account_id") }}'
+      - name: date_day 
+        description: '{{ doc("date_day") }}' 
+      - name: account_id 
+        description: '{{ doc("account_id") }}' 
```

Comment:
Should we apply not_null tests to these fields?

Reply:
true, updated